kubetest/main.go

```diff
@@ -512,6 +509,16 @@ func prepareGcp(o *options) error {
 	if err := migrateGcpEnvAndOptions(o); err != nil {
 		return err
 	}
+	if o.provider == "gke" && o.gcpNodeImage != "" {
+		// TODO(kubernetes/test-infra#3307): Should disappear when
+		// "bash" deployment is gone.
+		if o.deployment == "bash" {
+			os.Setenv("KUBE_GKE_IMAGE_TYPE", o.gcpNodeImage)
+		}
+		// TODO(kubernetes/test-infra#3536): This is used by the
+		// ginkgo-e2e.sh wrapper.
+		os.Setenv("NODE_OS_DISTRIBUTION", o.gcpNodeImage)
```

Comment:
`NODE_OS_DISTRIBUTION=${KUBE_NODE_OS_DISTRIBUTION:-${KUBE_OS_DISTRIBUTION:-gci}}`

isn't this will always use `KUBE_NODE_OS_DISTRIBUTION` or `KUBE_OS_DISTRIBUTION` or `gci` to overwrite `NODE_OS_DISTRIBUTION`? Or I read it wrong?